### ISSUE TITLE
-RigName is missing from startsniffin_Ahashpool_Bitcoin.bat

### ISSUE THREAD

**Issue by borzaka** [Sniffdog/Sniffdogminer#197]
And probably from `startsniffin_Zergpool_Bitcoin.bat` and `startsniffin_Zergpool_Litecoin.bat` too.

From readme:
> Add a -RigName for your mining rig for Ahashpool.com, Hashrefinery.com, Zpool.ca, and zergpool.com

So, why -RigName isn't in that .bat by default?

**Comment by Sniffdog** [comment]
Yes....copied from Ahashpool.... all fixed now... thanks for the catch
